brickstrap.sh

```diff
@@ -57,6 +58,18 @@ Options
                This option is required and should occur exactly once.
                Values are either a path to the project directory or the name of
                an example project shipped with brickstrap by default.
+-Q <qemu>      Specify which QEMU binary to use for emulation of foreign
+               instruction sets; this may be used to override auto-detected
+               QEMU.
+
+               The <qemu> value must be either the path to a binary or
+               the name of a Debian or QEMU architecture or the special string
+               'native'. If <qemu> corresponds to a binary it is used
+               unconditionally, without further validation. If <qemu> is an
+               architecture which matches the host architecture or 'native'
```

Comment:
Would this make more sense to say `none`here rather than `native` to get the idea across that qemu is not used in this case?